include/deal.II/lac/la_parallel_vector.templates.h

```diff
@@ -635,6 +720,47 @@ namespace parallel
 
     template <typename Number>
     void
+    Vector<Number>::import(const ReadWriteVector<Number>                  &V,
+                           VectorOperation::values                         operation,
+                           std_cxx11::shared_ptr<const CommunicationPatternBase> communication_pattern)
+    {
+      // If no communication pattern is given, create one. Otherwise, use the
+      // given one.
+      std_cxx11::shared_ptr<const Utilities::MPI::Partitioner> comm_pattern;
```

Comment:
There could also be a case where the communication pattern is not given but was already created and can be reused. I don't remember if it is what I did or not.